src/main/java/fr/iut/pathpilotapi/routes/dto/PositionDTO.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * PositionDTO.java                                 11 Dec 2024
+ * IUT de Rodez, no author rights
+ */
+
+package fr.iut.pathpilotapi.routes.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import java.util.Objects;
+
+/**
+ * Class representing a position with coordinates
+ */
+@Getter
+@Setter
+@RequiredArgsConstructor
+public class PositionDTO {
+
+    @NotNull
+    @Schema(description = "latitude value of the position")
+    private double latitude;
+
+    @NotNull
+    @Schema(description = "position value of the position")
+    private double longitude;
+
+    /**
+     * constructor
+     * @param longitude
+     * @param latitude
+     */
+    public PositionDTO(double longitude, double latitude) {
```

Comment:
Is this useful since there is the @RequiredArgsConstructor?

Reply:
Cannot delete it, due to the integration tests who need a constructor